<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(153, 229, 229);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>567</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Generate a Standard Strength Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>567</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Roman&quot;;</string>
   </property>
   <property name="text">
    <string>(8 characters long including a number, a special character and an uppercase letter)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_standard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Arial&quot;;
background-color: rgb(220, 126, 44);</string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_standard">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_strong">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>248</y>
     <width>567</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Roman&quot;;</string>
   </property>
   <property name="text">
    <string>(12 characters long with multiple numbers, special characters and uppercase letters)

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_strong">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Arial&quot;;
background-color: rgb(220, 126, 44);</string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>567</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Generate an Extra Strong Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_super">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>567</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Roman&quot;;</string>
   </property>
   <property name="text">
    <string>(20 characters long with multiple numbers, special characters and uppercase letters)

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_super">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Arial&quot;;
background-color: rgb(220, 126, 44);</string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>567</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Generate a Super Strong Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
